java/mvnd/Portfile

```diff
@@ -32,21 +32,20 @@ long_description mvnd aims at providing faster Maven builds using techniques kno
 homepage        https://github.com/apache/maven-mvnd
 supported_archs x86_64 arm64
 
-master_sites    https://downloads.apache.org/maven/mvnd/${version}/
+master_sites    https://github.com/apache/maven-mvnd/releases/download/${version}/
```

Comment:
Is this now a candidate to use the `github` portgroup?

Reply:
I believe it'll change back to the downloads.apache.org domain again for the next release, so I'd prefer to keep it like this for now.